### PR TITLE
CASSANDRA-16383 - merge cassandra-test back to master

### PR DESCRIPTION
**Pull request by aholmberg**

### CODE REVIEW

**Review by michaelsembwever — APPROVED** [review]
All CI looks good.

File: ccmlib/cluster.py [hunk 1]
```diff
@@ -412,7 +537,7 @@ def start(self, no_wait=False, verbose=False, wait_for_binary_proto=True,
         else:
             for node, p, mark in started:
                 try:
-                    timeout=kwargs.get('timeout', int(os.environ.get('CCM_CLUSTER_START_DEFAULT_TIMEOUT', 60)))
+                    timeout=kwargs.get('timeout', DEFAULT_CLUSTER_WAIT_TIMEOUT_IN_SECS)
```
Comment (tlasica) [inline ccmlib/cluster.py:540]: @aholmberg is it not a breaking change?

Comment (michaelsembwever) [reply ccmlib/cluster.py:540]: The `DEFAULT_CLUSTER_WAIT_TIMEOUT_IN_SECS ` variable is defined in a similar manner, except the default timeout has gone from one to two minutes.

ref: https://github.com/riptano/ccm/pull/723/files/51df6c6460275b1d52473a3b04fba8231cbb9921#diff-22e530c78060505b50ee68372e5bb5af00c039f96745a5975bef974d00473c6fR29

Comment (tlasica) [reply ccmlib/cluster.py:540]: yeah, all good indeed.

**Pull request merged**